autogen/convert_constants.py

```
@@ -63,19 +97,28 @@ def saw_constant(identifier, inIfBlock):
         seen_constants.append(identifier)
         return False
 
+def eval_constant(filename, constants, ident, val):
+    try:
+        set_to_val = int(eval(val, {}, {}))
```

Ruff (0.16.3)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)